bindings/go/main.go

```diff
@@ -209,11 +211,16 @@ func BlobToKZGCommitment(blob *Blob) (*KZGCommitment, error) {
 	if !loaded {
 		panic("trusted setup isn't loaded")
 	}
+	if blob == nil {
```

Comment:
Why so defensive? If caller sends in nil, he should not be surprised if a nil deref happens, imo

Reply:
I would rather be safe than sorry. It's a very simple check, might as well.